.golangci.yml

```
@@ -0,0 +1,44 @@
+    presets:
+      - common-false-positives
+issues:
+  max-issues-per-linter: 0
+  max-same-issues: 0
+formatters:
+  enable:
+    - gofumpt
+    - goimports
+  exclusions:
+    generated: lax
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)